mteb/tasks/STS/kor/KlueSTS.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+
+class KlueSTS(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="KLUE-STS",
+        dataset={
+            "path": "klue",
+            "name": "sts",
+            "revision": "349481ec73fff722f88e0453ca05c77a447d967c",
+        },
+        description="",
+        reference="https://arxiv.org/abs/2105.09680",
+        type="STS",
+        category="s2s",
+        eval_splits=["validation"],
+        eval_langs=["kor-Hang"],
+        main_score="cosine_spearman",
+        date=None,
```

Comment:
Could we please add a rough estimate? e.g. ("2021-01-01", "{paper date}")

Reply:
I've added an estimated dates based on the KLUE and Kor-STS papers. :)